% Written by Ava Hoffman
% Please use as you like, but it would be nice if you credited me :)
% 
% Compile with LaTeX.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass{ats_resume}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%	SPACING
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% size of the lefthand column: 
\newcommand{\experienceboxsize}{180mm}
% general spacing out of items
\newcommand{\itemspace}{\vspace{-2mm}}
% top header on 2nd page
\newcommand{\pgtwoheader}{\vspace{0.75in}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%	CONTENTS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%----------------------------------------------------------------------------------------
%	 PERSONAL INFORMATION
%----------------------------------------------------------------------------------------
\name{AVA HOFFMAN}
\jobtitle{Data Scientist}

\email{avamariehoffman@gmail.com}
\phone{(804) 687-7476}
\linkedin{/in/avahoffman}
\github{avahoffman}
\website{www.avahoffman.com}

\printheader

\vspace{9mm}


%----------------------------------------------------------------------------------------
%	 EXPERIENCE
%----------------------------------------------------------------------------------------
\addexperience{

	% Note that this work was conducted at Johns Hopkins University from 05/2021 - 06/2022.
        \experienceitem{Fred Hutch Cancer Center / Johns Hopkins University}{Senior Staff Scientist}{May 2021 - present}{Seattle, WA (remote)}{
%            \item Built outreach software for publishing content, previewing data, and helping scientists comply with NIH policies using Shiny, GitHub Actions, and other tools
%            \item Co-lead, analyzed, and curated data for \textcolor{special}{\href{https://biodigs.org}{BioDIGS}}, a soil microbe genetics research consortium of over 100 students and faculty 
%            \item Co-lead outreach and user research for the NIH \textcolor{special}{\href{https://anvilproject.org}{AnVIL Platform}}, enabling genomic data science via cloud computing for 500+ users
%            \item Led and managed \$579k NIH-funded program (\textcolor{special}{\href{https://genomicseducation.org}{"GEMs"}}) that designs open-source genomic data science training modules with under-resourced colleges
%            \item Led, developed, and managed \$200k/yr \textcolor{special}{\href{https://daseh.org}{Data Science for Environmental Health program}}, educating ~40 trainees and public health government professionals annually
%            \item Taught graduate-level courses in R programming and community data science consulting
            
%            This position is a data science role focused on building educational software and analyzing education outcomes among various biomedical research programs. I (1) Led and managed $579k NIH-funded education program ('GEMs') that designs, evaluates, and collects data on open-source genomic data science training modules with under-resourced colleges; (2) Led, developed, and managed $200k/yr Data Science for Environmental Health program (DaSEH), collecting data on learning modalities and educating 40 trainees and public health government professionals annually; (3) Co-lead outreach and user research analysis for the NIH AnVIL Platform, including providing technical support, enabling biomedical data science via cloud computing for 500+ users; (4) Built outreach software for publishing content, previewing data, and helping scientists create documentation with R, GitHub Actions, and other tools; (5) Developed interactive planning dashboards with R Shiny, improving NIH policy compliance for NIH scientists; (5) Co-lead, analyzed, and standardized data for the BioDIGS soil microbiome project, processing 5+ TB of genomic sequencing and soil property data across 300+ samples and 100+ students and faculty and making it available through online portals; (6) Taught graduate-level courses in R programming and community data science consulting; (7) Worked daily using tools like R, Excel, MS Office. 
            
            \item Led user research for NIH AnVIL Platform through survey design and interview analysis, identifying key feature priorities to increase platform adoption
            \item Developed interactive data planning dashboards with R Shiny, improving NIH policy compliance for NIH scientists
            \item Co-developed and deployed automated content publishing software using GitHub Actions and R that reduced manual content publishing time processing time by 50\%
            \item Designed and implemented statistical analyses for BioDIGS soil microbiome project, processing 5+ TB of genomic sequencing and soil property data across 300+ samples and 100+ students and faculty
            \item Secured and managed \$579k NIH grant through data-driven proposal development, creating genomic data science curriculum among 6 faculty at 4 institutions
            \item Established the \$200k/yr Data Science for Environmental Health program, teaching and evaluating data science pedagogy using repeated measures ML approaches
            \item Taught R programming courses focusing on data wrangling, reproducibility, and machine learning applications in public health
       }{hutch} \\
       
       \itemspace

       \experienceitem{Johns Hopkins University}{Postdoctoral Fellow}{Mar 2020 -  May 2021}{Baltimore, MD}{
%            \item Used bioinformatic tools to quantify genomic patterns among 1000+ plants adapted to 6 city landscapes
            \item Developed and implemented custom bioinformatic pipelines in R and Python to analyze 1TB+ of genomic data from 1,000+ plant samples
            \item Implemented evolutionary isolation models to understand genetic differences among distinct city environments
            \item Visualized complex genomic relationships using dimensional reduction techniques and genetic structure plots, leading to a publication in preparation
       }{jhu} \\
       
       \itemspace
       
       \experienceitem{Boston Consulting Group}{Data Scientist}{Mar 2019 - Mar 2020}{Boston, MA}{
            \item Productionalized PySpark pipeline describing and modeling client's 20K+ commercial banking customers for better product recommendations \& risk intervention, using logistic regression and random forest methods
            \item Created interactive RShiny dashboard integrating multiple data sources for carbon emissions tracking, enabling leadership to explore strategies for reducing internal carbon emissions by 30\% in 5 years
            \item Optimized and clustered client's vehicle inventory with Python and k-means clustering approach, improving confidence in leadership's nationwide product delivery decisions

        }{gamma} \\
       
        \itemspace
        
        \experienceitem{Colorado State University}{Researcher}{Aug 2013 - Mar 2019}{Fort Collins, CO}{
            %\item Led the Blue Grama Diversity Project, implementing hierarchical Bayesian models and analyzing 9,000+ genomic features to understand plant traits from population genetics
            \item Led statistical analysis for Blue Grama Diversity Project, implementing hierarchical Bayesian models in R to analyze 9,000+ genomic markers across 300+ geospatially distributed plant samples
            \item Developed the first custom computational pipelines for next-gen sequencing analysis in several species, improving reproducibility and future research capabilities
            \item Created reproducible analysis workflows using R Markdown and version control
        }{nifa} \\

}

%----------------------------------------------------------------------------------------
%	 EDUCATION
%----------------------------------------------------------------------------------------
\addeducation{
        	\educationitem{PhD}{Ecology}{Colorado State University}{05/2019}{Fort Collins, CO}{csu} \\
	\itemspace
        	\educationitem{BS}{Biology}{University of Virginia}{05/2012}{Charlottesville, VA}{uva}
}

%----------------------------------------------------------------------------------------
%	 TECHNICAL SKILLS
%----------------------------------------------------------------------------------------
\vspace{8mm}
\addtechnicalskills{
	% Python and packages
	%\technicalskillitem{\item Programming languages}
	\technicalskillitem{Programming languages}{Python, R} \\
	\technicalskillitem{Python packages}{scikit-learn, pandas, NumPy, SciPy, pytest, statsmodels, seaborn, matplotlib, Jupyter} \\ % Bokeh, Gensim
	\technicalskillitem{R packages}{tidyverse, RStan, shiny, bioconductor, ggplot2 + add ons, rMarkdown/bookdown, RStudio} \\ % leaps, lavaan, segmented, dplyr, reshape2, sva, vegan, bayesplot, ggrepel, gridExtra, semPlot
	\technicalskillitem{Version control}{Git, GitHub, Bitbucket} \\
	\technicalskillitem{Statisics}{Bayesian methods, multivariate statistics, permutational methods, dimensionality reduction, repeated measures} \\ % hiearchical, mixture models, anova, permanova
	\technicalskillitem{Supervised learning}{time series, random forest, linear/nonlinear/logistic regression, LDA, SEM} \\
	\technicalskillitem{Unsupervised learning}{PCA, k-means/hierarchical clustering, feature engineering} \\
	\technicalskillitem{Cloud technologies}{GCP, GitHub Actions} \\
	\technicalskillitem{Other technologies}{Docker, PySpark, SQL, APIs, QGIS, Tableau, Alteryx, SAS} % bash scripting
}


%----------------------------------------------------------------------------------------
%	 ACHIEVEMENTS
%----------------------------------------------------------------------------------------
\vspace{5mm}
\vspace{5mm}
\addachievements{

        \achievementitem{Excellence in Teaching Award}{2022 - 2025 (7x)}{Johns Hopkins University}{\faStar}
        
        \achievementitem{29 peer-reviewed publications}{2010-2025}{\textcolor{special}{\href{https://scholar.google.com/citations?user=k6RyLHsAAAAJ&hl=en}{Google Scholar}}}{\faBook}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\cheekyfooter{Résumé built using \textcolor{special}{\href{https://github.com/avahoffman/CV-and-resumes}{LaTeX}} \today.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{document} 
